node/src/components/small_network.rs

```diff
@@ -502,11 +507,9 @@ where
 
                 info!(our_id=%self.our_id, %peer_id, %peer_address, "established incoming connection");
                 // The sink is only used to send a single handshake message, then dropped.
-                let (mut sink, stream) = framed::<P>(transport).split();
-                let handshake = Message::Handshake {
-                    network_name: self.network_name.clone(),
-                    public_address: self.public_address,
-                };
+                let (mut sink, stream) =
+                    framed::<P>(transport, self.chain_info.maximum_net_message_size).split();
```

Comment:
IMO it's worth mentioning this in the PR description - that we're allowing configuration of the max framed size for messages.

Reply:
Hmm, technically it's not configuration, but a chain spec value? It's mentioned in the linked PR https://github.com/CasperLabs/casper-node/pull/1256

Reply:
I'll add it though.